packages/angular/projects/dev/src/app/vertical-nav/direct-icon/vertical-nav-icon.demo.html

```diff
@@ -20,15 +20,15 @@
       </form>
       <div class="settings">
         <a href="javascript://" class="nav-link nav-icon">
-          <clr-icon shape="cog"></clr-icon>
+          <cds-icon shape="cog"></cds-icon>
         </a>
       </div>
     </header>
     <div class="content-container">
       <clr-vertical-nav [clrVerticalNavCollapsible]="true" [(clrVerticalNavCollapsed)]="collapse">
         <ng-container *ngFor="let item of case.items">
           <a clrVerticalNavLink href="javascript:void(0)" *ngIf="!item['children']">
-            <clr-icon [attr.shape]="item.icon" *ngIf="item['icon']" clrVerticalNavIcon></clr-icon>
+            <cds-icon [attr.shape]="item.icon" *ngIf="item['icon']" clrVerticalNavIcon></cds-icon>
```

Comment:
Is `[attr.shape]` still necessary? I was under the impression that it was not.

Reply:
PR is still in progress. It is a Draft one. I will go through every single `cds-icon` attribute tomorrow so I remove the unnecessary usages or replace them with the proper ones. For example, the one thing I did today was to replace `class="is-solid"` with `solid` property.

Reply:
@mathisscott  The `[attr.shape]` is needed because if we do`[shape]` the angular compiler says the `[shape] input` doesn't exist. Another way to doing this is `shape="{{ icon name }}"` but I believe the `[attr.shape]` is much better.